itou/www/apply/tests/tests_submit.py

```diff
@@ -46,7 +46,7 @@
 class ApplyTest(TestCase):
     def test_anonymous_access(self):
         siae = SiaeFactory(with_jobs=True)
-        job_seeker = JobSeekerFactory()
```

Comment:
Je ne comprend pas pourquoi on a besoin de faire cela ?! (et pareil pour tout le fichier)

Reply:
Je le fais pour dégager la factory `JobSeekerProfile` qui ajoute sinon une manière de faire qui n'est pas "consacrée". On veut que les gens passent toujours par la `JobSeekerFactory`  avec son `post_generation` sinon cela fait deux points d'entrée, profondément différents.

Reply:
Je ne te suis pas :-/
Là on est sur une `JobSeekerFactory` et pas une `JobSeekerProfileFactory`

Reply:
Ah, OK, vu. C'est en effet une erreur, j'ai fait un search-and-replace qui a été trop gros. On peut ignorer ceux-ci je les corrige de suite.

Reply:
Corrigé.